### DISCUSSION TITLE
Making Joomla more user friendly, decrease ID in backend.

### DISCUSSION THREAD

**Discussion by pl71** [joomla/joomla-cms#38776] [General]
### Steps to reproduce the issue
Administrator - Content - Articles

### Expected result
Less complicated table.

### Actual result
What is the reason for showing the ID column (and sorting by same)?
It's not a bug, but I see no reason to have this column shown by default.
How many users are searching by ID? Are you have such usage statistics?
As a side note - the popup window allowing to select visible columns has no separate close button.

### System information (as much as possible)
Joomla 4.2.2

**Comment by brianteeman** [comment]
Apart from it being that way for 15 years the main reason is that it ensures the latest created content is at the top of the list. Imagine having a thousand articles and the default sort order is title alphabetical. Create a new article and save and close. Oh whoops I forgot something - now where is that article so I can open it again.

**Comment by pl71** [comment]
For the same reason, I'm using "Date Modified" which is placing the latest article on top. Or maybe I'm missing something? Anyway ID could be useful but is not so intuitive. I'm not sure how ID-s are assigned, if they can be re-used, etc...
Even more - I would like "Date Modified (descending)" to be the first column, not the last.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38770">issues.joomla.org/tracker/joomla-cms/38770</a>.</sub>

**Comment by HLeithner** [comment]
Just a personal note, the ID column is one of the most important for me, for user support. Since you can filter for it (simple use ID:123 in the textfield) it a fast why to communicated with users to be in the same page.

**Comment by alikon** [comment]
moving to discussion as this is clearly not a bug

**Comment by chmst** [comment]
In Joomla 4.2 you can hide the column if you don't want to see it, and other columsn too.
